csharp/src/Google.Protobuf.Test/Collections/RepeatedFieldTest.cs

```diff
@@ -599,5 +600,61 @@ public void IList_Insert()
             list.Insert(1, "middle");
             CollectionAssert.AreEqual(new[] { "first", "middle", "second" }, list);
         }
+
+        [Test]
+        public void ToString_Integers()
+        {
+            var list = new RepeatedField<int> { 5, 10, 20 };
+            var text = list.ToString();
+            Assert.AreEqual("[ 5, 10, 20 ]", text);
```

Comment:
Not entirely sure. Will open a new issue to check, but merge this code - if we need to change anything, that will be in code that hasn't changed in this PR anyway.